### ISSUE TITLE
Questions about the Script

### ISSUE THREAD

**Issue by aminum005** [olakunlevpn/PonziPedia-V1.5#5]
Hello,

I have a few questions to ask, tried contacting you via WhatsApp & Facebook but got no response from you.

Firstly, I'd love to know if the script is for sale or free.

Also, is it possible to integrate Wordpress as frontend to handle the pages, with the ponzipedia script dashboard working at the backend.

Can I use custom @font-face for the script font?

Finally, kindly provide a medium I can contact you with.

Thank you.

**Comment by Kfastharealkimg** [comment]
How do i remove the maintainance mode?

**Comment by nweze4life** [comment]
The guy is a very busy person, reaching him on phone is difficult, notwithstanding keep trying ok or you can chat me on +2348069843807

**Comment by dalesf** [comment]
> How do i remove the maintainance mode?

Is your referral link working well, like instead of activation fees been paid to admin is being paid to the person that referred you

**Comment by olakunlevpn** [comment]
> > How do i remove the maintainance mode?
> 
> Is your referral link working well, like instead of activation fees been paid to admin is being paid to the person that referred you

Yes everything work perfectly and we have include some additional features on the new release which allow you to choose who will receive the referral fees between admin and the referral from the `app/config/app.php`
